scripts/00spinup.py

```diff
@@ -57,23 +61,18 @@
 # Initialization
 # ------------------------------------------
 # Arrays for total values (across all layers), with time as the first axis
-subsurface_storage = np.zeros((nt,))
-surface_storage = np.zeros((nt,))
-wtd = np.zeros((nt, nx, ny))
-et = np.zeros((nt,))
-overland_flow = np.zeros((nt, nx, ny))
+subsurface_storage = np.zeros(nt)
+surface_storage = np.zeros(nt)
+wtd = np.zeros((nt, ny, nx))
+et = np.zeros(nt)
+overland_flow = np.zeros((nt, ny, nx))
 
 # ------------------------------------------
 # Loop through time steps
 # ------------------------------------------
 for i, (pressure_file, saturation_file) in enumerate(zip(pressure_files, saturation_files)):
-    pressure = pfio.pfread(pressure_file)
-    saturation = pfio.pfread(saturation_file)
-
-    # We set pressure/saturation values outside our mask to np.nan to indicate missing values.
-    # Individual functions that deal with these arrays are written to handle np.nan values properly.
-    pressure[mask == 0] = np.nan
-    saturation[mask == 0] = np.nan
+    pressure = metadata.pfb_data(pressure_file)
+    saturation = metadata.pfb_data(saturation_file)
 
     # total subsurface storage for this time step is the summation of substorage surface across all x/y/z slices
     subsurface_storage[i, ...] = np.sum(
```

Comment:
I know I said I wanted to provide `total_*` functions in addition to the gridded versions, but on second thoughts I found this more clear and explicit. Especially for me since I'm new to all this, having to do the np.sum over the correct axes manually made me think (and document) what I was dealing with (2d vs 3d). This also avoids, by extension, the problem of summing over time, or samples over time (which may be scenarios we deal with soon), leaving it to the script to do as it likes.

So I've left it this way for now. If you feel strongly about it I can add the `total_*` functions.